classes/book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'rental'
+
+# class book
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] This method suppose to create a new rental not push a rental in an array.
[Hint] It's supposed to  be like that:
```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
 end
```